<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="Lista">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>831</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Orden">
    <attribute name="title">
     <string>Ordenes</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>560</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Asignar a:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>560</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Precio Servicio ($):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveOrder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>700</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AgregarEspecial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>640</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar SE</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="PrecioServicio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Precio insumos ($):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Resumen de orden:</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>261</width>
       <height>281</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>259</width>
        <height>279</height>
       </rect>
      </property>
      <widget class="QLabel" name="servicio0">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>161</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="servicio1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>161</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="servicio2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>161</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="servicio3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>161</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="servicio4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>161</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="stockbutton0">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>41</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stockbutton1">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>41</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stockbutton2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>41</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stockbutton3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>41</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stockbutton4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>220</y>
         <width>41</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>110</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>210</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <zorder>frame_6</zorder>
      <zorder>frame_5</zorder>
      <zorder>frame_3</zorder>
      <zorder>servicio0</zorder>
      <zorder>stockbutton0</zorder>
      <zorder>servicio1</zorder>
      <zorder>stockbutton1</zorder>
      <zorder>servicio2</zorder>
      <zorder>stockbutton2</zorder>
      <zorder>servicio3</zorder>
      <zorder>stockbutton3</zorder>
      <zorder>servicio4</zorder>
      <zorder>stockbutton4</zorder>
     </widget>
    </widget>
    <widget class="QTextEdit" name="PrecioInsumo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="NombreServicio">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Servicio:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="LectorPuesto">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>610</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Puesto 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Puesto 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Puesto 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Puesto 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Brooklyn</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Servicios:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Eliminar">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>640</y>
       <width>341</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>660</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>M Pago:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="LectorMetPago">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>710</y>
       <width>191</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Efectivo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nequi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Datafono</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transferencia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Daviplata</string>
      </property>
     </item>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>261</width>
       <height>431</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="AgregarLista">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>20</y>
        <width>241</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLabel" name="MostrarServicios">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="ListaServicios">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="NuevaOrden">
     <property name="geometry">
      <rect>
       <x>613</x>
       <y>640</y>
       <width>101</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Nueva orden</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>560</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Total a pagar:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Total">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>570</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>570</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>90</y>
       <width>461</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>90</y>
       <width>481</width>
       <height>461</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame_4</zorder>
    <zorder>frame</zorder>
    <zorder>label_8</zorder>
    <zorder>label_6</zorder>
    <zorder>SaveOrder</zorder>
    <zorder>AgregarEspecial</zorder>
    <zorder>PrecioServicio</zorder>
    <zorder>label_5</zorder>
    <zorder>label_11</zorder>
    <zorder>scrollArea</zorder>
    <zorder>PrecioInsumo</zorder>
    <zorder>NombreServicio</zorder>
    <zorder>label_4</zorder>
    <zorder>LectorPuesto</zorder>
    <zorder>label_10</zorder>
    <zorder>Eliminar</zorder>
    <zorder>label_12</zorder>
    <zorder>LectorMetPago</zorder>
    <zorder>frame_2</zorder>
    <zorder>NuevaOrden</zorder>
    <zorder>label_3</zorder>
    <zorder>Total</zorder>
    <zorder>label_13</zorder>
   </widget>
   <widget class="QWidget" name="Administrador">
    <attribute name="title">
     <string>Administrador</string>
    </attribute>
    <widget class="QPushButton" name="LogInButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>84</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>55</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Admin:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AdminName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AdminPass">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>55</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Desde:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>150</y>
       <width>55</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Hasta</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GenerarInforme">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>430</y>
       <width>711</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Generar Informe</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="BeginTime">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>391</width>
       <height>240</height>
      </rect>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="EndTime">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>180</y>
       <width>391</width>
       <height>240</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
